sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIO.java

```diff
@@ -653,10 +654,14 @@ public CreateTransaction withTimestampBound(TimestampBound timestampBound) {
   @AutoValue
   public abstract static class Write extends PTransform<PCollection<Mutation>, PDone> {
 
+    private long maxNumMutations;
```

Comment:
This variable should not be needed - this is an AutoValue class

Reply:
Done